src/bindings/python/tests/test_torchvision_to_ov/test_preprocessor.py

```diff
@@ -0,0 +1,224 @@
+import numpy as np
+import copy
+import os
+import pytest
+from PIL import Image
+
+import torch
+import torch.nn.functional as F
+import torchvision.transforms as transforms
+
+from openvino.runtime import Core
+
+from openvino.preprocess.torchvision_to_ov import PreprocessConverter
+
+
+class Convnet(torch.nn.Module):
+    def __init__(self, input_channels):
+        super(Convnet, self).__init__()
+        self.conv1 = torch.nn.Conv2d(input_channels, 6, 5)
+        self.conv2 = torch.nn.Conv2d(6, 16, 3)
+
+    def forward(self, x):
+        x = F.max_pool2d(F.relu(self.conv1(x)), 2)
+        x = F.max_pool2d(F.relu(self.conv2(x)), 2)
+        return x
+
+
+def _infer_pipelines(test_input, preprocess_pipeline, input_channels=3):
+    torch_model = Convnet(input_channels)
+
+    torch.onnx.export(torch_model, torch.randn(1, input_channels, 224, 224), "test_convnet.onnx", verbose=False, input_names=["input"], output_names=["output"])
+    core = Core()
+    ov_model = core.read_model(model="test_convnet.onnx")
+    os.remove("test_convnet.onnx")
```

Comment:
do you have any test via PyTorch Frontend?

cc, @mvafin

Reply:
I agree that with covert_model api it can be even more straightforward: `ov_model = mo.convert_model(pt_model, example_input=input_dict)`

Reply:
Running `convert_model` is time consuming, even for such a small model - for these tests it increases total test time (on my machine of course) from 5 to 22 seconds. It's going to add up in precommit, while the biggest benefit would probably a bit cleaner code. What do you think?

Reply:
To me it sounds really strange that export to `onnx` + conversion by ONNX FE spends much more time than just conversion PT FE. We have multiple (hundreds) PT FE layer tests that do not consume much time.

The scenario through `convert_model` is most likely way for PT models and more important way where we are going now. So I am leaning to test this one in the first order.

Reply:
I refactored the tests into using `mo.convert_model`.